packages/core/tocco-ui/src/EditableValue/EditorProvider.spec.js

```diff
@@ -1,15 +1,15 @@
-import {mount} from 'enzyme'
+import {screen, render, fireEvent} from '@testing-library/react'
 
 import EditorProvider from './EditorProvider'
 
 describe('tocco-ui', () => {
   describe('EditableValue', () => {
     describe('typeFormatterProvider', () => {
       test('should render a type and set props', () => {
-        const wrapper = mount(<EditorProvider componentType="string" value="test" />)
+        render(<EditorProvider componentType="string" value="test" />)
 
-        expect(wrapper.find('StringEdit')).to.have.length(1)
-        expect(wrapper.find('StringEdit').props().value).to.eql('test')
+        expect(screen.getByTestId('string')).exist
```

Comment:
question(blocking): Is this really a helpful assertion?

I prefer to remove `data-testid` on the `EditorProvider` and remove this assertion since it does not tell us if the EditorProvider really choosed the right component.
In the old test it checked if the mapping was working and the `StringEdit` was rendered.

How about using this assertion instead:

```
expect(screen.getByRole('input')).to.exist
```